frontend/src/pages/admin/AdminManageUsersPage.js

```diff
@@ -0,0 +1,124 @@
+import React, { Component } from 'react';
+import {
+  Container,
+  Row,
+  Col,
+  Form,
+  Input,
+  Button,
+  Modal,
+  ModalHeader,
+  ModalBody,
+  ModalFooter,
+  Alert
+} from 'reactstrap';
+import AdminNavigator from '../../components/AdminNavigator';
+import '../../styles/admin/AdminNavigator.scss';
+import '../../styles/admin/AdminHome.scss';
+import { upgradeUser } from '../../utils/api';
+
+class AdminManageUsersPage extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      modal: false,
+      username: '',
+      alert: null
+    };
+  }
+
+  toggleModal = () => {
+    this.setState({
+      modal: !this.state.modal
+    });
+  };
+
+  setStateAlert = message => {
+    this.setState({
+      alert: message
+    });
+  };
+
+  handleUpgrade = async () => {
+    const query = this.state.username.trim();
+
+    const {
+      success,
+      result: { results },
+      message
+    } = await upgradeUser({
+      user_username: query
+    });
+
+    if (success) {
+      this.toggleModal();
+      this.setStateAlert(message);
+    } else {
+      this.toggleModal();
+      this.setStateAlert(message);
+    }
+  };
+
+  render() {
+    let header;
+    if (this.state.alert !== null) {
+      header = <Alert color="danger">{this.state.alert}</Alert>;
+    }
+    return (
+      <Container fluid>
+        <Row>
+          <Col lg="2">
+            <AdminNavigator />
+          </Col>
+          <Col lg="10">
+            <h1>Upgrade an Account</h1>
+            <hr />
+            {header}
+            <h5>Make an accout an admin account.</h5>
```

Comment:
oops, typo in accouNt

Reply:
how about changing this to "Grant admin privileges to an existing account."

Reply:
:+1: